beacon-chain/db/filesystem/cache.go

```diff
@@ -55,7 +57,7 @@ var _ BlobStorageSummarizer = &blobStorageCache{}
 
 func newBlobStorageCache() *blobStorageCache {
 	return &blobStorageCache{
-		cache: make(map[[32]byte]BlobStorageSummary, params.BeaconConfig().MinEpochsForBlobsSidecarsRequest*fieldparams.SlotsPerEpoch),
+		cache: make(map[[32]byte]BlobStorageSummary),
```

Comment:
Why removing the capacity of the map?

Reply:
I was thinking that due to the way map expansion works (more heuristic than lists, based on bin averages), even with this hint it may usually wind up needing to expand anyway. But the hint won't hurt anything, might as well keep it.